Apps/WebClient/src/ClientApp/test/.eslintrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+    rules: {
+        "no-empty-function": "off",
```

Comment:
are we having nested linting configurations?, should we move this up to the root level?

Reply:
Only for the test, we want to be able to have an empty function on stubs and mocks but not on the ClientApp